docs/manifest-types.md

```diff
@@ -357,13 +358,78 @@ This type of manifest adds a navigation entry into the navigation.xml.
                 "app-catalog-gui": {
                     "label": "Apps",
                     "title": "Apps",
+                    "bundle": "app-catalog-gui",
                     "icon": "fa-archive",
                     "module": "app-catalog-gui",
                     "controller": "index",
                     "action": "index"
                 }
             },
             "after": "users"
+        },
+        {
+            "navigations": {
+                "main": {
+                    "pages": {
+                        "main-nested": {
+                            "bundle": "main",
+                            "controller": "index",
+                            "action": "index",
+                            "visible": "1"
+                        }
+                    }
+                }
+            },
+            "before": "before-main"
+        }
+    ]
+}
+```
+
+### Unwire Navigation Manifest
+
+This type of manifest removes a navigation entry into the navigation.xml.
+
+```json
+{
+    "unwire-navigation": [
+        {
+            "navigations": {
+                "delete": null
+            },
+            "after": "main"
+        },
+        {
+            "navigations": {
+                "catalog": {
+                    "pages": {
+                        "price-product-schedule": {
+                            "icon": null,
+                            "pages": {
+                                "price-product-schedule-dry-run-import": {
+                                    "icon": null
```

Comment:
same here

Reply:
we decided to use 'null' in unwire-manifest as a flag that given field should be deleted

Reply:
Doesnt that make it a bit cumbersome in PHP with isset()?
Couldnt we use a more descriptive key/value pair as in

    remove => ...
    
or is it easier the way above?

Reply:
@dereuromark can you please prepare an example for the case
`"unwire-navigation": [
            {
                "navigations": {
                    "main": {
                        "title": null,
                        "action": null
                    }
                }
            }
        ],`

Reply:
You need to use proper code fencing
I fixed your comment.

As for your question

whats the purpose of this config?
If you want to unwire a certain element, why not using a descriptive key for it?
Would be simpler and less "obscure" I imagine.

Reply:
It is not a blocker for now, I was just wondering about a more user friendly way of defining removes

Reply:
the idea was that in general `null` means `no value`, so we remove this attribute or element where null is provider. 

If we don't need it, then we also don't need `remove` key, because `unwire-navigation` means `remove` by itself, there is no other logic for this type of manifest.

Everything is much easier by example. Here is an example of navigation file: 

```
<?xml version="1.0"?>
<config>
    <main>
        <label>Main</label>
        <title>Main</title>
        <icon>fa-archive</icon>
        <bundle>main</bundle>
        <controller>index</controller>
        <action>index</action>
        <pages>
            <main-nested>
                <label>Before main - nested</label>
                <title>Before main - nested</title>
                <icon>fa-archive</icon>
                <bundle>before-main-nested</bundle>
                <controller>index</controller>
                <action>index</action>
                <visible>0</visible>
            </main-nested>
        </pages>
    </main>
</config>
``` 

Here is current manifest:

```
"unwire-navigation": [
    {
        "navigations": {
            "main": {
                "pages": {
                    "main-nested": {
                        "visible": null,
                        "icon": null
                    }
                }
            }
        }
    }
],
```

This manifest will remove 2 attributes `visible` and `icon` in given structure and we'll get the result:

```
<?xml version="1.0"?>
<config>
    <main>
        <label>Main</label>
        <title>Main</title>
        <icon>fa-archive</icon>
        <bundle>main</bundle>
        <controller>index</controller>
        <action>index</action>
        <pages>
            <main-nested>
                <label>Before main - nested</label>
                <title>Before main - nested</title>
                <bundle>before-main-nested</bundle>
                <controller>index</controller>
                <action>index</action>
            </main-nested>
        </pages>
    </main>
</config>
```
Have I got it right, that the suggestion is not to use `null`, so that the new manifest will look like: 

```
"unwire-navigation": [
    {
        "navigations": {
            "main": {
                "pages": {
                    "main-nested": {
                        "visible",
                        "icon"
                    }
                }
            }
        }
    }
],
```
?

Reply:
OK, after discussion, using list style will complicate with nested levels.
So we can move forward with assoc special null style.